openwisp_controller/config/migrations/0011_add_relevant_permissions_to_admin_and_operators.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations 
+from django.models import Permission 
+
+
+def assignPerm(apps, schema_editor): 
+    Group= apps.get_model('openwisp_user', 'Group')
+    admin = Group.objects.get(name="Administrator")
+    operator= Group.objects.get(name="Operator")
+    operators_and_admins_can_change=["device", "config", "template", "location", "floorplan",]
+    operators_read_only_admins_manage=["vpn", "ca", "certificate",]
+    manage_operations=["add", "change", "delete"]
+
+    for i in operators_and_admins_can_change:

```

Comment:
@nemesisdesign correct me if I'm wrong, won't it be more efficient to ```set``` all the permissions at once after collecting them all in a list?

Reply:
That won't really affect performance at all. The function call overhead might actually *decrease* performance in this case.

Reply:
That won't really affect performance at all. The function call overhead might actually *decrease* performance in this case.